### PR TITLE
Add competition details email

### PR DESCRIPTION
**Pull request by RealOrangeOne**

### CODE REVIEW

File: SB2018/2018-04-17.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+I've attached a map of the event, it will take place in:
+
+> Building 42
+> University Rd
+> Southampton
+> SO17 1BJ
+
+Also on the map is the recommended car park. This car park is free on weekends.
+
+All of these details are mirrored on our website: https://sourcebots.co.uk/events/2018/competition/.
```
Comment (Adimote) [inline SB2018/2018-04-17.md]: If we're not actually putting the information on the website for this, the last sentence should be worded differetnyl

**Review by Adimote — DISMISSED** [review]

File: SB2018/2018-04-17.md (hunk 1 above)
Comment (Adimote) [reply SB2018/2018-04-17.md]: as in, "You can see all information on the website:"

Comment (RealOrangeOne) [reply SB2018/2018-04-17.md]: All this information _is already_ on the website, besides parking map

**Review by Adimote — APPROVED** [review]
There's a bit of incontinuity between Kajetan's section and ours, but it's not worth fixing. :shipit:

**Comment by Adimote** [comment]
OH we should mention the marathon

**Pull request merged**